sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_generated/v7_2_preview/aio/operations/_role_definitions_operations.py

```diff
@@ -23,7 +25,7 @@ class RoleDefinitionsOperations:
     instantiates it for you and attaches it as an attribute.
 
     :ivar models: Alias to model classes used in this operation group.
-    :type models: ~key_vault_client.models
+    :type models: ~azure.keyvault.v7_2.models
```

Comment:
It breaks nothing and doesn't need to change, but I'm curious why there are so many docstring references assuming a nonexistent package for 7.2 🤔